code/script.js

```diff
@@ -229,97 +231,191 @@ const setSecret = () => {
 
 // This function to start (and restart) the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
+// Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  setSecret()
+  generateBoard()
+  selectQuestion()
+
+  console.log(secret)
```

Comment:
console.log can be removed when project is finished :)

Reply:
That right, I will remove it :)